CHANGES.rst

```diff
@@ -46,6 +46,8 @@ Unreleased
     available in a template before using it. Test functions can be
     decorated with ``@environmentfunction``, ``@evalcontextfunction``,
     or ``@contextfunction``. :issue:`842`, :pr:`1248`
+-   Support for ``pgettext`` and ``npgettext`` (only short form syntax).
+    :issue:`441, 793`, :pr:`1126`
```

Comment:
```suggestion
    :issue:`441`, :issue:`793`, :pr:`1126`
```

Otherwise the clickable links won't work properly

Reply:
This is actually fine as is.

Reply:
However, it should only be 441, the other issue is a dupe with no discussion, and we typically don't link PRs that have linked issues.

Reply:
Yep, there are examples with this form of syntax

```
-   Fix behavior of ``loop`` control variables such as ``length`` and
    ``revindex0`` when looping over a generator. :issue:`459, 751, 794`,
    :pr:`993`
```

But I can change to suggested one